packages/cdktf/lib/terraform-module.ts

```diff
@@ -4,13 +4,13 @@ import { deepMerge } from "./util";
 
 export interface TerraformModuleOptions {
   readonly source: string;
-  readonly version: string;
+  readonly version?: string;
```

Comment:
This isn't a required property for using modules.

Reply:
👍